### PR TITLE
Add a couple more projects to the whitelist

### PR DESCRIPTION
**Pull request by rhiever**
Looking to add a couple more of my projects to the whitelist. Thank you @bboe!

### CODE REVIEW

File: update_checker_app/controllers.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from .models import Installation, Package, PythonVersion
 
 
-ALLOWED_PACKAGES = {'lazysusan', 'praw', 'redditanalysis', 'tpot'}
+ALLOWED_PACKAGES = {'lazysusan', 'praw', 'redditanalysis', 'tpot', 'xrff2csv', 'datacleaner'}
```
Comment (bboe) [inline update_checker_app/controllers.py]: Can you keep these sorted please?

**Comment by rhiever** [comment]
Done.

BTW: What does the update checker do if the person does not have an internet connection?

**Comment by bboe** [comment]
https://github.com/bboe/update_checker/blob/master/update_checker.py#L123

There's a 1-second timeout. If the request isn't completed in that time then it assumes there is no update unless the package version is the permanent cache and has been in there less than an hour, then it could still report there is an update.

**Comment by bboe** [comment]
I'll update the app sometime this evening.

**Comment by rhiever** [comment]
Thanks @bboe! My only worry was that the update check would prevent the
script from running if it wasn't online, but that doesn't seem to be the
case.

On Wednesday, March 2, 2016, Bryce Boe notifications@github.com wrote:

> I'll update the app sometime this evening.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/bboe/update_checker_app/pull/4#issuecomment-191422475
> .

## 

Randal S. Olson, Ph.D.
Postdoctoral Researcher, Institute for Biomedical Informatics
University of Pennsylvania

E-mail: rso@randalolson.com | Twitter: @randal_olson
https://twitter.com/randal_olson
http://www.randalolson.com

**Comment by bboe** [comment]
App updated.

**Pull request merged**